rest/data/patch.go

```diff
@@ -79,6 +79,36 @@ func (pc *DBPatchConnector) FindPatchById(patchId string) (*restModel.APIPatch,
 	return &apiPatch, nil
 }
 
+func (pc *DBPatchConnector) FindPatchesByIds(patchIds []string) ([]restModel.APIPatch, error) {
+	mgobsonIds := []mgobson.ObjectId{}
```

Comment:
Naming this variable by just its type is confusing. I would name it something like "patchObjectIDs". Alternatively you can make your patch.ByIds function take in an array of strings and do the conversion there